utilities/getAgoByDate.ts

```diff
@@ -0,0 +1,24 @@
+export default function getAgoByDate(date: Date): [number, string, string] {
```

Comment:
Nice implementation. However, it would be better if this functions is split into two: long expression (for desktop) and short expression (for mobile).

Reply:
Probably I prefer such name 🤔

```suggestion
export default function getRelativeDateString(date: Date): [number, string, string] {
```